lxd/images.go

```diff
@@ -106,7 +106,7 @@ func unpack(file string, path string) error {
 	if err != nil {
 		shared.Debugf("Unpacking failed")
 		shared.Debugf(string(output))
-		return err
+		return fmt.Errorf("Unpack failed, %s.", err)
```

Comment:
> Do we have verbose on where we shouldn't?

I don't think so.  On my system tar complains about every single failure once the disk runs out of space, i.e. output has hundreds of lines complaining about "not enough space".  I'm hesitant to capture only the tail of the output in case the original failure was somehow different than subsequent failures.